var/spack/repos/builtin/packages/py-pygobject/package.py

```diff
@@ -41,8 +41,3 @@ class PyPygobject(Package):
     depends_on('gobject-introspection')
 
     patch('pygobject-2.28.6-introspection-1.patch')
-
-    def install(self, spec, prefix):
-        configure("--prefix=%s" % prefix)
-        make()
-        make("install", parallel=False)
```

Comment:
So I'm the one who got this race, and actually the `libelf` version hasn't changed much since I wrote this 3 years ago, so I think it's still a problem.  I believe it is *just* a race during install, though.